js/jsonld-signatures.js

```diff
@@ -782,7 +782,7 @@ function _getDataToHash(input, options) {
     if(options.domain !== null && options.domain !== undefined) {
       toHash += '@' + options.domain;
     }
-  } else if(options.algorithm === 'LinkedDataSignature2015') {
+  } else if(options.algorithm === 'LinkedDataSignature2015' || options.algorithm === 'EcdsaKoblitzSignature2016') {
```

Comment:
This check is going to become increasingly problematic as we add signature suites. @dlongley should we have a "supportedSignatureSuites" value in some constant such that we don't have to keep updating this section of the code?